test/common/dailies.coffee

```diff
@@ -49,9 +50,15 @@ newUser = (addTasks=true)->
       user.ops.addTask {body: {type: task, id: shared.uuid()}}
   user
 
+getTasksForUser = (user) ->
+  return user.habits.concat(user.dailys)
+          .concat(user.todos).concat(user.rewards)
+
 cron = (usr, missedDays=1) ->
   usr.lastCron = moment().subtract(missedDays,'days')
-  usr.fns.cron()
+  daysMissed = usr.fns.shouldCronRun()
+  if daysMissed isnt 0 
```

Comment:
@crookedneighbor shouldCronRun initially returned true or false but then I moved it to return numbers so it shouldn't be the problem. The problem is either with js inheritance or in cron itself but since I didn't change anything in cron the second option seems a bit strange